tests/scr-support_scrolled-doc/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en" lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<script>
+			/**
+			 * A simple change that proves that scripting is enabled.
+			 * 
+			 */ 
+			function test() {
+			  const div_no_scripting = document.getElementById('no_scripting');
+			  div_no_scripting.style.display = 'none';
+			  const div_scripting = document.getElementById('scripting');
+			  div_scripting.style.display = 'block';
+			}
+			window.addEventListener('load', test);
+		</script>
+		<title>rendition:flow set to scrolled-doc when scripting is supported</title>
```

Comment:
This is a fine title but it's the *same* title as the other test, so please rename one of them.

IMO this is correct for the current test, but not properly describing the other test. I'm not actually sure which of the two SHOULDs in https://w3c.github.io/epub-specs/epub33/rs/#confreq-rs-scripted-scrolled the other test is meant to correspond to, but:

- if the first SHOULD, I think the other title should be "Reading system supports spine-level scripting with scrolled-doc"
- if the second SHOULD, I think the other title should be "rendition:flow set to scrolled-continuous when scripting is supported"

Reply:
Actually... the other tests' titles were wrong! It should have been scrolled-continuous and not scrolled-doc... (changed now)